tests/seta/conftest.py

```diff
@@ -0,0 +1,129 @@
+                "build_system_type": "buildbot",
+                "job_type_name": "Reftest e10s",
+                "platform": "linux32",
+                "platform_option": "opt",
+                "ref_data_name": "Ubuntu VM 12.04 mozilla-inbound opt test reftest-e10s-1",
+            }, {
+                "build_system_type": "buildbot",
+                "job_type_name": "Build",
+                "platform": "osx-10-7",
+                "platform_option": "opt",
+                "ref_data_name": "OS X 10.7 mozilla-inbound build",
+            }, {
+                "build_system_type": "taskcluster",
+                "job_type_name": "desktop-test-linux32/opt-reftest-e10s-1",
+                "platform": "linux32",
+                "platform_option": "opt",
+                "ref_data_name": "desktop-test-linux32/opt-reftest-e10s-1",
+            }, {
+                "build_system_type": "taskcluster",
+                "job_type_name": "desktop-test-linux64/opt-reftest-e10s-2",
+                "platform": "linux64",
+                "platform_option": "opt",
+                "ref_data_name": "desktop-test-linux64/opt-reftest-e10s-2",
+            }
+        ]
+    }
+
+
+@pytest.fixture
+def task_request_stored(test_repository):
+    # test_repository reprents the repository test_treeherder_jobs
+    tr = TaskRequest(
+        repository=test_repository,
+        counter=1,
+        last_reset=timezone.now(),
+        reset_delta=DEFAULT_TIMEOUT,
+    )
+    tr.save()
+    return tr
+
+
+@pytest.fixture
+def sanitized_data(runnable_jobs_data):
+    return _sanitize_data(runnable_jobs_data)
+
+
+@pytest.fixture
+def all_job_priorities_stored(job_priority_list):
+    '''Stores sample job priorities
+
+    If you include this fixture in your tests it will guarantee
+    to insert job priority data into the temporary database.
+    '''
+    JobPriority().store_job_priority_data(job_priority_list)
+    return job_priority_list
+
+
+@pytest.fixture
+def job_priority_list(sanitized_data):
+    jp_list = []
+    for datum in sanitized_data:
+        jp_list.append(JobPriority(
+            testtype=datum['testtype'],
+            buildtype=datum['platform_option'],
+            platform=datum['platform'],
+            buildsystem=datum['build_system_type'],
+            priority=1,
+            timeout=0,
+        ))
+        # Mark a TC job as low priority
+        if datum['testtype'] == 'reftest-e10s-2':
+            jp_list[-1].priority = DEFAULT_LOW_PRIORITY
+            jp_list[-1].timeout = DEFAULT_TIMEOUT
+        # Mark a BB job as low priority
+        if datum['testtype'] == 'web-platform-tests-1':
+            jp_list[-1].priority = DEFAULT_LOW_PRIORITY
+            jp_list[-1].timeout = DEFAULT_TIMEOUT
+
+    return jp_list
+
+
+@pytest.fixture
+def jp_index_fixture(job_priority_list):
+    return job_priority_index(job_priority_list)
+
+
+@pytest.fixture
+def failures_fixed_by_commit():
+    return {u'you look like a man-o-lantern': [
```

Comment:
The "man o lantern" quip is just a corny in-joke from one of the treeherder developers (can't remember who...). I think we can leave it in.

I don't think it really matters that we still have b2g-related test data here, these are just unit tests. I'd rather spend effort on improving other things than generating new test data that is functionally no different from what we have now.